test/system/220-healthcheck.bats

```diff
@@ -80,7 +80,6 @@ EOF
 Status           | healthy
 FailingStreak    | 0
 Log[-1].ExitCode | 0
-Log[-1].Output   |
```

Comment:
Had to retrofit the bad tests which expects no output from health-check. While this PR fixes that.
Not checking log output here since we are already doing that in `e2e` just avoiding redundancy.

Reply:
Please also check the output here

Reply:
Please no. This is not "redundancy". e2e tests run only in CI; system tests run on real live actual systems, before packages make it to RHEL or Fedora. They are our last-ditch effort to catch problems.

Reply:
@edsantiago Thanks added a new test for this. But i had to modify `jq` output so it contains newline character rather than actual newline. Could you review `bats` part please.